src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -88,21 +96,37 @@ export const ProgressStep = ({ ...props }) => {
 
   return (
     <li className={classes}>
-      {currentSvg || completeSvg || incompleteSvg}
-      <ProgressStepLabel className={`${prefix}--progress-label`}>
-        {label}
-      </ProgressStepLabel>
-      {componentsX &&
-      secondaryLabel !== null &&
-      secondaryLabel !== undefined ? (
-        <p className={`${prefix}--progress-optional`}>{secondaryLabel}</p>
-      ) : null}
-      <span className={`${prefix}--progress-line`} />
+      <div
+        style={{
+          display: 'inline-flex',
```

Comment:
Wondering what this style is for, but if this style is absolutely necessary, could you please create a PR to https://github.com/IBM/carbon-components/pulls with the corresponding style ruleset?